latest/Dockerfile.amd64

```diff
@@ -29,6 +29,7 @@ RUN apt-get update && apt-get install -y \
     git \
     wget \
     nano \
+    ffmpeg \
```

Comment:
This package would be helpful for debugging client GUI tests (recording video of the test execution).
I am trying to implement on this PR https://github.com/owncloud/client/pull/9961